t/ui/10-tests_overview.t

```diff
@@ -28,7 +28,7 @@ my $test_case   = OpenQA::Test::Case->new;
 my $schema_name = OpenQA::Test::Database->generate_schema_name;
 my $schema      = $test_case->init_data(
     schema_name   => $schema_name,
-    fixtures_glob => '01-jobs.pl 02-workers.pl 04-products.pl 05-job_modules.pl'
+    fixtures_glob => '01-jobs.pl 02-workers.pl 04-products.pl 05-job_modules.pl 06-job_dependencies.pl'
```

Comment:
I think I have suggested the same recently to Martchus: it's better if we can limit the fixtures we need in individual test modules. So *maybe* we are better off with loading dependencies fixtures in a dedicated test module only testing the UI features of dependencies. Or you only add relevant data in the sub-tests further down where needed

Reply:
Of course if she really just needs the two rows inserted by that file it would be fine, right? This fixture file is really minimal and determined to a certain purpose.

Reply:
Yes, that could be. I merely wanted to nudge us into making that an explicit decision. So if we agree on "no, we want it this way" then it is ok as well :)

Reply:
Just like Marchus said, this fixture file is really minimal,  so I prefer to keep it. is it ok?

Reply:
yes, that's ok